index.js

```diff
@@ -0,0 +1,32 @@
+/*canvas-background*/
+
+document.addEventListener("DOMContentLoaded", function() {
+    const canvas = document.getElementById("myCanvas");
+    const ctx = canvas.getContext("2d");
+
+/*functions*/
+
+    function drawBall(ctx, centerX, centerY, radius, fillColor) {
+        ctx.beginPath();
+        ctx.arc(centerX, centerY, radius, 0, 2 * Math.PI);
+        ctx.fillStyle = "pink"; 
```

Comment:
fillcolor seems to be passed in arguments but not used here

Reply:
Good catch, removing that